### ISSUE TITLE
Errors and NaNs running the Quick Start tutorial

### ISSUE THREAD

**Issue by crypdick** [nautechsystems/nautilus_trader#1054]
# Bug Report

### Expected Behavior
Quick start should work end-to-end without errors.

### Actual Behavior
Running the backtest produces the following output (contains nans):

```
2023-03-20T12:52:49.652227856Z [INF] BACKTESTER-001.BacktestEngine: Unrealized PnLs:
2023-03-20T12:52:49.652240194Z [INF] BACKTESTER-001.BacktestEngine: None
[...]
2023-03-20T12:52:49.654190696Z [INF] BACKTESTER-001.BacktestEngine:  Returns Statistics
2023-03-20T12:52:49.654191956Z [INF] BACKTESTER-001.BacktestEngine: -----------------------------------------------------------------
2023-03-20T12:52:49.666270281Z [INF] BACKTESTER-001.BacktestEngine: Returns Volatility (252 days):  nan
2023-03-20T12:52:49.666287825Z [INF] BACKTESTER-001.BacktestEngine: Average (Return):               -7.166082929481977e-05
2023-03-20T12:52:49.666289956Z [INF] BACKTESTER-001.BacktestEngine: Average Loss (Return):          -7.166082929481977e-05
2023-03-20T12:52:49.666291408Z [INF] BACKTESTER-001.BacktestEngine: Average Win (Return):           nan
2023-03-20T12:52:49.666292976Z [INF] BACKTESTER-001.BacktestEngine: Sharpe Ratio (252 days):        nan
2023-03-20T12:52:49.666294320Z [INF] BACKTESTER-001.BacktestEngine: Sortino Ratio (252 days):       -15.874507866387544
2023-03-20T12:52:49.666314412Z [INF] BACKTESTER-001.BacktestEngine: Profit Factor:                  0.0
2023-03-20T12:52:49.666315675Z [INF] BACKTESTER-001.BacktestEngine: Risk Return Ratio:              nan
```

### Steps to Reproduce the Problem

1. Run the [Quick Start example](https://docs.nautilustrader.io/getting_started/quick_start.html)

### Specifications

- OS platform: Ubuntu 22.10
- Python version: py39
- `nautilus_trader` version: `__latest__`

**Comment by crypdick** [comment]
In addition, the tutorial has you run `!curl https://raw.githubusercontent.com/nautechsystems/nautilus_data/main/scripts/hist_data_to_catalog.py | python -`

which contains the import:

`from nautilus_trader.backtest.data.providers import TestInstrumentProvider`

which should be:

`from nautilus_trader.test_kit.providers import TestInstrumentProvider`

**Comment by cjdsellers** [comment]
Hi @crypdick 

Thanks so much for reporting these issues. I've updated the `nautilus_data` repo to the latest version of `nautilus_trader` and fixed the imports. I've also modified the quickstart strategy to produce more trades, so the backtest performance results are much healthier now.

The NaNs weren't actually an error, but merely indicate the statistic can't calculate a reasonable value based on the data provided (not enough trades).

I'll be doing some more work on the docs today. In the mean time I hope this helps you out!
